Makefile

```diff
@@ -789,6 +796,17 @@ keystone_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_de
 	make keystone_cleanup
 	make mariadb_cleanup
 
+.PHONY: cinder_kuttl_run
+cinder_kuttl_run: ## runs kuttl tests for the keystone operator, assumes that everything needed for running the test was deployed beforehand.
+	INSTALL_YAMLS=${INSTALL_YAMLS} kubectl-kuttl test --config ${CINDER_KUTTL_CONF} ${CINDER_KUTTL_DIR}
+
+.PHONY: cinder_kuttl
+cinder_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy mariadb_deploy_validate cinder_deploy_prep keystone ## runs kuttl tests for the cinder operator. Installs openstack crds and cinder operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
```

Comment:
It's missing `keystone_deploy` and `cinder` as dependencies.  I would also recommend leaving `mariadb_validate` for last, so meanwhile the assert waits for the mariadb database to be up, it has time to bring up the keystone and cinder operators and CRs

Reply:
I'm trying to deploy cinder in my crc, it seems that it needs `rabbitmq` and `rabbitmq_deploy` as well. But even with that it's not coming up, you should probably ping someone from the dev team to get a definitive list of requirements